tests/unittests/OnnxImporterTest.cpp

```diff
@@ -807,6 +807,34 @@ TEST(onnx, gather) {
   EXPECT_TRUE(gather->getResult().dims().equals({2, 4, 2}));
 }
 
+/// Test loading GatherRanges from an ONNX model.
+TEST(onnx, gatherranges) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  std::string netFilename("tests/models/onnxModels/gatherranges.onnxtxt");
+  auto *F = mod.createFunction("main");
+  Placeholder *output;
+  Tensor data(ElemKind::FloatTy, {6});
+  Tensor ranges(ElemKind::Int32ITy, {2, 2, 2});
+
+  {
+    ONNXModelLoader onnxLD(netFilename, {"data", "ranges"},
+                           {&data.getType(), &ranges.getType()}, *F);
+    output = EXIT_ON_ERR(onnxLD.getOutputByName("output"));
+  }
+  EE.compile(CompilationMode::Infer, F);
```

Comment:
why do we need to compile if not running?
Testing just loader part should be fine here and testing Op itself in the OperatorTest.